src/components/TranslationLeaderboard.js

```diff
@@ -0,0 +1,245 @@
+    display: block;
+    font-size: ${(props) => props.theme.fontSizes.s};
+    opacity: 0.6;
+  }
+`
+
+const LangaugeContainer = styled(TextContainer)`
+  display: inherit;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    display: none;
+  }
+`
+
+const StyledEmoji = styled(Emoji)`
+  display: block;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    display: none;
+  }
+`
+
+const TranslationLeaderboard = () => {
+  const leaderboardData = {
+    monthData,
+    quarterData,
+    allTimeData,
+  }
+  const [filterAmount, updateFilterAmount] = useState(10)
+  const [dateRangeType, updateDateRangeType] = useState("monthData")
+
+  const showLess = () => {
+    updateFilterAmount(10)
+  }
+
+  const showMore = () => {
+    updateFilterAmount(50)
+  }
+
+  return (
+    <Content>
+      <ButtonRow>
+        <ButtonSecondaryStyled
+          onClick={() => updateDateRangeType("monthData")}
+          isSelected={dateRangeType === "monthData"}
+        >
+          <Translation id="page-contributing-translation-program-acknowledgements-translation-leaderboard-month-view" />
+        </ButtonSecondaryStyled>
+        <ButtonSecondaryStyled
+          onClick={() => updateDateRangeType("quarterData")}
+          isSelected={dateRangeType === "quarterData"}
+        >
+          <Translation id="page-contributing-translation-program-acknowledgements-translation-leaderboard-quarter-view" />
+        </ButtonSecondaryStyled>
+        <ButtonSecondaryStyled
+          onClick={() => updateDateRangeType("allTimeData")}
+          isSelected={dateRangeType === "allTimeData"}
+        >
+          <Translation id="page-contributing-translation-program-acknowledgements-translation-leaderboard-all-time-view" />
+        </ButtonSecondaryStyled>
+      </ButtonRow>
+      <Table>
+        <Header>
+          <ItemNumber>#</ItemNumber>
+          <TextContainer>
+            <Translation id="page-contributing-translation-program-acknowledgements-translator" />
+          </TextContainer>
+          <LangaugeContainer>
+            <Translation id="page-contributing-translation-program-acknowledgements-language" />
+          </LangaugeContainer>
+          <WordsContainer>
+            <Translation id="page-contributing-translation-program-acknowledgements-total-words" />
+          </WordsContainer>
+        </Header>
+        {leaderboardData[dateRangeType].data
+          .filter((item, idx) => idx < filterAmount)
+          .map((item, idx) => {
+            const { user, translated, languages } = item
+            let emoji = null
+            if (idx === 0) {
+              emoji = ":trophy:"
+            } else if (idx === 1) {
+              emoji = ":2nd_place_medal:"
+            } else if (idx === 2) {
+              emoji = ":3rd_place_medal:"
+            }
+            return (
+              <Item key={idx}>
+                {emoji ? (
+                  <Emoji mr={"1rem"} text={emoji} />
```

Comment:
These trophy emojis feel way too small - can we make them look similar to the bug bounty leaderboard?
![Image 2021-11-29 at 7 52 11 PM](https://user-images.githubusercontent.com/8097623/143971379-7cf4f3cf-a68f-4078-8ffd-1974721fd534.jpg)

Reply:
I know we're listing more results but in general I feel like this table could use a bit more breathing room, similar to size & spacing of the bug bounty leaderboard rows.